nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md

```diff
@@ -0,0 +1,36 @@
+---
+title: NServiceBus fails during performance counter setup
+summary: Sometimes an NServiceBus endpoint hangs or fails during startup while initializing the performance counters due corrupt performance counter libraries
+tags: []
+---
+
+## Symptons
+
+Sometimes an NServiceBus endpoint hangs or fails during startup while initializing the performance counters at the following location `NServiceBus.Unicast.Transport.Monitoring.ReceivedPerformanceDiagnostics.SetupCounter`.
+
+## Cause 
+
+This can happen when the performance counter libraries get corrupted.
+
+## Resolution
+
+The performance counters libraries need to be rebuild by doing the following steps:
+
+1. Open an elevated command prompt
+2. Execute the following command rebuilds the performance counter libraries:
+
+    `LODCTR /R`  (Mind the uppercase /R)
+
```

Comment:
3 and 4 are not realky steps. Just two extra points if the above two steps dont work.